core/tests/webdriverio_utils/ExplorationEditorSettingsTab.js

```diff
@@ -157,6 +166,7 @@ var ExplorationEditorSettingsTab = function() {
       explorationLanguageInput, 'Language input takes too long to be visible.');
     expect(await explorationLanguageInput.$('option:checked').getText()).
```

Comment:
Why didn't you drop the `option:checked` part here like you did for the protractor test? Ditto above for `expectFirstStateToBe`

Reply:
done